democracy_club/templates/base.html

```diff
@@ -56,7 +56,7 @@
         <aside class="ds-status messages" aria-label="Status">
             <ul class="ds-stack">
                 {% for message in messages %}
```

Comment:
These messages are from the [messages framework](https://docs.djangoproject.com/en/4.2/ref/contrib/messages/) and show up on first page load after some event, for example signing up to the mailing list.

Removing that will remove any other message we currently show.

As you're altering the base template, I think this would be better just hard coded. That way it will show on all pages too (unless that's not the intent?)